src/sentry/static/sentry/app/components/button.tsx

```diff
@@ -24,7 +24,7 @@ type Props = {
   busy?: boolean;
   to?: string | object;
   href?: string;
-  icon?: string;
+  icon?: string | object;
```

Comment:
Does ts get mad if we be more strict with the type and change `object` to `React.Element`?

Reply:
Yes it does get mad